plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableProperties.java

```diff
@@ -67,6 +67,7 @@ public class HiveTableProperties
     public static final String CSV_SEPARATOR = "csv_separator";
     public static final String CSV_QUOTE = "csv_quote";
     public static final String CSV_ESCAPE = "csv_escape";
+    public static final String PARQUET_BLOOM_FILTER_COLUMNS = "parquet_bloom_filter_columns";
```

Comment:
Pls document the new table property in `hive.md` to make users aware in the documentation about it.
https://trino.io/docs/current/connector/hive.html#table-properties

Can be done in a separate PR if there are no pending comments.

Reply:
Let's handle doc update in this PR @jkylling

Reply:
Added docs for the new table property. Also corrected the documentation for the corresponding ORC table property which I believe was incorrect.